db/migrations/0002_alter_guild_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-11-18 18:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='guild',
+            name='description',
+            field=models.TextField(null=True),
```

Comment:
Ensure consistency in the `description` field's nullability across all migration files. In the initial migration, the `description` field was set to `blank=True`, but here it is set to `null=True`. Consider whether both `blank=True` and `null=True` are needed for your use case.